### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Documentation and source comment typos found with `codespell -q 3 -w --skip="./help,./po,*.svg" -I ../pitivi--whitelist.txt` 
where whitelist contained:
```
iff
que
ressources
seeked
superceded
unselect
```

### CODE REVIEW

**Comment by luzpaz** [comment]
I know this isn't the right place for PRs but I was hoping you could still consider merging this

File: pitivi/utils/pipeline.py [hunk 1]
```diff
@@ -613,7 +613,7 @@ def simple_seek(self, position):
 
     def _busMessageCb(self, bus, message):
         if message.type == Gst.MessageType.ASYNC_DONE:
-            self.commiting = False
```
Comment (aleb) [inline pitivi/utils/pipeline.py]: Oops! Many thanks!

**Comment by luzpaz** [comment]
all good ?

**Comment by luzpaz** [comment]
@aleb please weigh in...thanks

**Comment by aleb** [comment]
Looks good, I'll merge it later today. Thanks!

Any suggestions on how we can integrate this in our development workflow?

**Comment by luzpaz** [comment]
@aleb check out https://github.com/lucasdemarchi/codespell/issues/139
though I've not figured out how to implement this in a manner that meets my standards as satisfactory. Partly because FreeCAD doesn't have a webhook solution to do so.

File: pitivi/viewer/move_scale_overlay.py [hunk 2]
```diff
@@ -408,7 +408,7 @@ def __set_size(self, size):
         self.handles[(Edge.bottom, Edge.left)].position = numpy.array([0, size[1]])
         self.handles[(Edge.bottom, Edge.right)].position = numpy.array([size[0], size[1]])
         self.handles[(Edge.top, Edge.right)].position = numpy.array([size[0], 0])
-        self.__update_egdes_from_corners()
+        self.__update_edges_from_corners()
```
Comment (luzpaz) [inline pitivi/viewer/move_scale_overlay.py]: please also review these changes...

File: pitivi/viewer/move_scale_overlay.py [hunk 3]
```diff
@@ -426,7 +426,7 @@ def __update_handle_sizes(self):
         for handle in self.handles.values():
             handle.restrict_radius_to_size(smaller_size)
 
-    def __update_egdes_from_corners(self):
+    def __update_edges_from_corners(self):
```
Comment (luzpaz) [inline pitivi/viewer/move_scale_overlay.py]: here..

File: pitivi/viewer/move_scale_overlay.py [hunk 4]
```diff
@@ -494,7 +494,7 @@ def on_motion_notify(self, cursor_pos):
         if self.__clicked_handle:
             # Resize Box / Use Handle
             self.__clicked_handle.on_drag(click_to_cursor)
-            self.__update_egdes_from_corners()
+            self.__update_edges_from_corners()
```
Comment (luzpaz) [inline pitivi/viewer/move_scale_overlay.py]: and here.

**Comment by aleb** [comment]
Looks fine. I can squash it.

File: pitivi/utils/pipeline.py (hunk 1 above)
Comment (aleb) [reply pitivi/utils/pipeline.py]: Hm, actually it seems to be a field which has never been used. I removed it in a separate commit.

**Comment by aleb** [comment]
Merged, thanks!

**Pull request closed without merging**